examples/computer_vision/fast.py

```diff
@@ -40,13 +40,13 @@ def fast_demo(console):
     else:
         img_color = af.load_image("../../assets/examples/images/man.jpg", True);
```

Comment:
Not sure how to correctly specify path to input images inside of assets repository.